devices/xiaomi.js

```diff
@@ -2394,6 +2497,8 @@ module.exports = [
             exposes.binary('window_open', ea.STATE, true, false),
             e.valve_detection().setAccess('state', ea.ALL),
```

Comment:
I see `schedule_options` was not added to exposes, can you  add that? Looks like you will need to use the `composite` exposes.